signac_dashboard/modules/video_viewer.py

```diff
@@ -35,7 +35,8 @@ def make_card(filename):
             return {'name': self.name + ': ' + filename,
                     'content': render_template(
                         self.template,
-                        poster=self.poster if job.isfile(
+                        jobid=jobid,
```

Comment:
shouldn't this be `jobid=job._id`

Reply:
It is the same (see the first line of the snippet below), but this could be cleaned up. Originally I had a reason for this distinction but it's no longer relevant (and I can't remember what the reason was). Feel free to make a PR! https://github.com/glotzerlab/signac-dashboard/blob/98fe791cc9a6d37386169586903fac9fc9466a64/signac_dashboard/modules/video_viewer.py#L31-L42